packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseSecurityRequirementObject-test.js

```diff
@@ -97,7 +97,7 @@ describe('Security Requirement Object', () => {
     expect(arr.get(0).element).to.equal('customOauth2');
     expect(arr.get(0).length).to.equal(1);
 
-    const scopes = arr.get(0).get('scopes');
+    const scopes = arr.get(0).get(0).value;
```

Comment:
Comment isn't specific to this line but for file (which GitHub don't let you comment on).

There are no tests for the case that the scheme is not registered in components. Thus when the scheme is not known the code may produce an invalid element:

```yaml
unknown: []
```

```yaml
element: unknown
```

----

Secondly, what happens if I call my element scheme the same as a pre-existing element?


```yaml
array: []
null: []
annotation: []
```

Reply:
The first part will be part of a different PR and the second part is already discussed above.